scopesim/effects/spectral_trace_list.py

```diff
@@ -471,8 +471,8 @@ class SpectralTraceListWheel(Effect):
     }
     _current_str = "current_trace_list"
 
-    def __init__(self, cmds=None, **kwargs):
-        super().__init__(cmds, **kwargs)
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
```

Comment:
Why did you undo these changes? It is better to make the `cmds` argument explicit and limit the use of `kwargs` to arguments you don't expect.

Reply:
Yep. I get that. But this ends in a bigger oroject of making sure every effect object has the explicit argument cmds in the init call